packages/app/src/lib/audit.ts

```diff
@@ -51,16 +51,6 @@ async function hasPasswordBeenCompromised(passwordHash: string) {
     return false;
 }
 
-function isItemExpiringOrExpired(expiryDate: Date) {
-    const thirtyDaysFromNow = add(new Date(), { days: 30 });
-
-    if (expiryDate <= thirtyDaysFromNow) {
-        return true;
-    }
-
-    return false;
-}
-
```

Comment:
I decided that only items should be reported that are actually expired, the reason being that it's possible to add an expiration period of less than 30 days (may be rare, but we don't know how people are going to use it) and in that case items would always appear in the security report, making it kind of pointless. If people want items to appear in the security report sooner, they can simply enter a shorter expiration period.